src/containers/Tenant/Diagnostics/TenantOverview/TenantOverview.scss

```diff
@@ -30,13 +32,15 @@
         align-items: center;
 
         margin-bottom: 10px;
+
+        line-height: 24px;
```

Comment:
This line-height also matches between the two blocks, but it is more of a coincidence than a rule. Decided not to put it in a variable